lib/perl/Genome/Model/Tools/DetectVariants2/Dispatcher.pm

```diff
@@ -102,6 +102,9 @@ class Genome::Model::Tools::DetectVariants2::Dispatcher {
         lsf_queue => {
             default_value => Genome::Config::get('lsf_queue_dv2_workflow'),
         },
+        lsf_resource => {
+            default => Genome::Config::get('lsf_resource_dv2_dispatcher'),
```

Comment:
I've changed the line that said `default_value` to match the other five that are easily seen in this PR. :smile: